jest-ci.json

```diff
@@ -0,0 +1,10 @@
+{
+  "moduleFileExtensions": ["js", "json", "ts"],
+  "rootDir": ".",
+  "roots": ["./src"],
+  "testEnvironment": "node",
+  "testRegex": [".*\\.spec\\.ts$", ".*\\e2e-spec\\.ts$"],
+  "transform": {
+    "^.+\\.(t|j)s$": "ts-jest"
+  }
+}
```

Comment:
I think it makes sense to move this file to `src/test`. This is where the E2E configuration is also stored.

Reply:
I tried it but I got some problems with the paths and the coverage reports, I will go a bit deeper with it to try to found a solution 👍🏻

Reply:
It should work if you set `"coverageDirectory": "../coverage"` 👍 

```json
{
  "moduleFileExtensions": ["js", "json", "ts"],
  "rootDir": ".",
  "roots": ["../src"],
  "testEnvironment": "node",
  "testRegex": [".*\\.spec\\.ts$", ".*\\e2e-spec\\.ts$"],
  "transform": {
    "^.+\\.(t|j)s$": "ts-jest"
  },
  "coverageDirectory": "../coverage"
}

```

Reply:
Scratch the above ☝️ Coverage is generated but it's empty.

If you can't find a solution right now to this I would not spend too much time on it. We can take a look another time into it.

Reply:
Changed in [0eeb45cd623c9ecc22644f8be9c13d3551685aac](https://github.com/5afe/safe-client-gateway-nest/pull/140/commits/0eeb45cd623c9ecc22644f8be9c13d3551685aac) 

Setting `coverageDirectory` did the trick 👍